third-party/eazy/v1/openapi.yaml

```diff
@@ -2235,30 +2235,30 @@ components:
           type: string
           example: 'Jane Mayer'
     autoMessageResponse:
-      description: Automated message response
+      description: An automated message response
       type: object
       properties:
         body:
-          description: Body/text of the message
+          description: The body/text of the message
           type: string
           example: 'Sorry, we are currently closed. Your question will be answered as soon as possible.'
         createdAt:
-          description: When message is created
+          description: When the message is created
           type: string
           format: date-time
           example: '2020-03-11T19:00:02.000000Z'
         enabled:
           description: Is the message enabled?
           type: boolean
         type:
-          description: Name of the assistant
+          description: The name of the assistant
```

Comment:
```suggestion
          description: The type of the auto message
```

Reply:
Ain't this used for automatic messages such as greeting and you are contacting us outside our business hours? If so, the assistant should not be here, right?